src/release.js

```diff
@@ -0,0 +1,59 @@
+const github = require('@actions/github');
+
+async function getLatestRelease(token) {
+  const octokit = github.getOctokit(token);
+  const { owner, repo } = github.context.repo;
+
+  const allReleasesResp = await octokit.request('GET /repos/{owner}/{repo}/releases', {
+    owner,
+    repo,
+  });
+
+  const latestRelease = (allReleasesResp && allReleasesResp.data && allReleasesResp.data.length)
```

Comment:
please check the docs about what you should expect here. this condition may be simplified as you can probably assume that you get something from the octokit request